modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/ImplicitTxContext.java

```diff
@@ -26,10 +26,12 @@
 
 /** Context that always creates implicit transaction. */
 public class ImplicitTxContext implements QueryTransactionContext {
-    public static final QueryTransactionContext INSTANCE = new ImplicitTxContext();
-
     private final HybridTimestampTracker observableTimeTracker = HybridTimestampTracker.atomicTracker(null);
 
+    public static ImplicitTxContext instance() {
```

Comment:
I think, the name `create` will better reflect semantic

Reply:
Done.